build/xtask/src/dist.rs

```diff
@@ -618,6 +618,13 @@ fn translate_srec_to_other_formats(
             &cfg.img_dir(image_name).join(format!("{}.{}", name, ext)),
         )?;
     }
+    // also generate a vmem, it is useful for running in RTL sims
+    // it is translated using `srec_cat` from the `.bin`
```

Comment:
Was this something that objcopy was unable to do?

Reply:
I tried the objcopy version, but was unable to get it to work (albeit, did not spend too much time on it).  The last few versions of binutils have been improving this feature though.  The Opentitan folks have been thinking the same thing as you, https://github.com/lowRISC/opentitan/issues/1107.  Looks like maybe it is fixed in the latest release?  I can give it a shot, but I would be hesitant to require a version of binutils that came out 2 weeks ago...

Reply:
Also, curious what brought you back to this, were you having trouble with `srecord`? or just looking to simplify the dependency list?